packages/wrangler/src/deployment-bundle/find-additional-modules.ts

```diff
@@ -49,16 +50,41 @@ export async function findAdditionalModules(
 			name: m.name,
 		}));
 
+	// Try to find a requirements.txt file
+	const isPythonEntrypoint =
+		getBundleType(entry.format, entry.file) === "python";
+
+	if (isPythonEntrypoint) {
+		try {
+			const pythonRequirements = await readFile(
+				path.resolve(entry.directory, "requirements.txt"),
+				"utf-8"
+			);
+
+			// This is incredibly naive. However, it supports common syntax for requirements.txt
+			for (const requirement of pythonRequirements.split("\n")) {
+				modules.push({
+					type: "python-requirement",
+					name: path.resolve(entry.moduleRoot, requirement),
```

Comment:
I guess this results in `/path/to/module/root/bcrypt==4.0.1`. Can you strip the operator and version from the requirements? The absolute path shouldn't be necessary either.

We probably just want to parse as long as we see a valid identifier (something like `^[^\d\W]\w*` should work). This should grab the package name even if it contains dashes and avoid reading in any operators and version numbers.

Reply:
Do we want to strip version specifiers? Pyodide seems to throw a relatively helpful error if you specify a version that isn't supported. e.g. `bcrypt==4.0.0` gives `ValueError: Can't find a pure Python 3 wheel for 'bcrypt==4.0.0'.`

Reply:
Ahhh, this doesn't wokr in production, so I'll strip these